### DISCUSSION TITLE
script

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6377] [Help]
Embed Component
why the script below does not work , knowing that it works well with visual studio
![Capture](https://user-images.githubusercontent.com/106697208/174487196-2a8e2c9c-298c-4faa-ad93-128068e083a6.PNG)

**Comment by melohagan** [comment]
Hi @rafikApp 

Does it work if you remove the html and body tags. E.g. just have
```
<script>
  function ...
</script>
<img src... />
```

**Comment by rafikApp** [comment]
unfortunately it doesn't work

**Reply by melohagan** [reply]
It may be worth having a look at this comment here: https://github.com/Budibase/budibase/discussions/5112#discussioncomment-2456819

**Reply by SantoshSrinivas79** [reply]
@melohagan any idea why none of the below methods work?

**Comment by insomniacj93** [comment]
@rafikApp 
Try removing every sintax that isn't a function or variable.
Also you're doing this in Handlebars, your code is Javascript so try the Javascript tab right next to it.

The Budibase javascript compiler, compiles the code at it's nude form.

Example, try using just this code in JavaScript tab (instead of Handlebars): 

function.greet() {
   windows.location = 'http://www.google.com';
 }
 setTimeout(greet, 6000);

The images you can easly use the element on top of the page.

**Comment by rafikApp** [comment]
Hi @insomniacj93 
to use the JavaScript tab, you have to write a function that returns a value, which is not the case for me. 
And I tried what you told me but it didn't work




![Capture1](https://user-images.githubusercontent.com/106697208/174980938-3c0cf108-ffd9-48fd-abbe-d3b373289cea.PNG)

**Comment by insomniacj93** [comment]
@rafikApp 
I understand that, but writing javascript in handlebars won't get us anywhere either, as you noticed the sintax isn't recognised by the compiler, we can find another way around using handlebars https://handlebarsjs.com/ . But honestly I'm not sure what you want to archive. You want the application to launch google search?

**Comment by rafikApp** [comment]
@insomniacj93 
google website is just an example.
I want to display an animation (gif), and when it finished I want to navigate to another page of the application

**Comment by insomniacj93** [comment]
@rafikApp 
Handlebars.js
https://handlebarsjs.com/guide/expressions.html#helpers
